manifests/f/Flow-Launcher/Flow-Launcher/1.12.1/Flow-Launcher.Flow-Launcher.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.2.1 using InputObject 🤖 $debug=QUSU.CRLF.7-2-8
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Flow-Launcher.Flow-Launcher
+PackageVersion: 1.12.1
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: user
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
+UpgradeBehavior: install
+ReleaseDate: 2023-01-25
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Flow-Launcher/Flow.Launcher/releases/download/v1.12.1/Flow-Launcher-Setup.exe
+  InstallerSha256: 4C3B46B196521ECD276A7860069697EF0685F4951086790182F2D2C88D2C73F5
```

Hi @jjw24, the correct hash for this .exe is `c00e02203f15381b55adb46e44e57e4ea00990f25d4b9425e5bb1d6650133883`.

```suggestion
  InstallerSha256: c00e02203f15381b55adb46e44e57e4ea00990f25d4b9425e5bb1d6650133883
```